partcad/src/partcad/context.py

```
@@ -284,25 +284,25 @@ def _get_project_recursive(self, project, import_list: list[str]):
-                #         or not imports[x]["onlyInRoot"],
-                #         imports,
+                #         lambda x: "onlyInRoot" not in dependencies[x]
+                #         or not dependencies[x]["onlyInRoot"],
+                #         dependencies,
                 #     )
-                #     imports = list(filtered)
-                for prj_name in imports:
-                    pc_logging.debug("Checking the import: %s vs %s..." % (prj_name, next_import))
+                #     dependencies = list(filtered)
+                for prj_name in dependencies:
+                    pc_logging.debug("Checking the dependency: %s vs %s..." % (prj_name, next_import))
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)